app/plugins/Wallet/css/wallet.css

```diff
@@ -115,9 +117,9 @@ select:hover {
 	height: calc(50% - 46px);
 }
 .list {
-	height: calc(100% - 108px);
+	height: calc(50%);
 	padding: 16px;
-	overflow:auto;
+	overflow:hide;
```

Comment:
Should keep the space after the colon in CSS since that's been the convention.

We could have a CSS linter fire for this.